test/import.bats

```diff
@@ -390,6 +390,53 @@ eigth:
     [ -f /dir/file2 ]
     [ -f /dir/files/file3 ]
 EOF
+    stacker build
+}
+
+@test "import with dir contents" {
+  mkdir folder1
+  touch folder1/file1
+  cat > stacker.yaml <<EOF
+first:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+  - path: folder1/
+    dest: /folder1/
+  run: |
+    [ -f /folder1/file1 ]
+
+second:
+  from:
+    type: built
+    tag: first
+  run: |
+    mkdir -p /folder1
+    touch /folder1/file1
+    touch /folder1/file2
 
+third:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+    - path: stacker://second/folder1/
+      dest: /folder1/
+  run: |
+    [ -f /folder1/file1 ]
+    [ -f /folder1/file2 ]
+
+fourth:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+  - path: folder1/
+    dest: /
+  run: |
+    ls -l /
+    [ -f /file1 ]
+EOF
     stacker build
 }
```

Comment:
And in case of

```
import:
  path: folder1/folder2/
  dest: folder3/
```

I would expect content of folder 2 to be cooied under folder3 right? Can we add a test? Existing tests use the same folder name in src and dest, or only '/' at dest.

And a similar test case for stacker:// import.

Reply:
Fixed